pkg/obs/mount.go

```diff
@@ -26,14 +26,14 @@ func sendCommand(cmd CommandRPC, mountClient http.Client) error {
 	if err != nil {
 		return err
 	}
-	log.Infof("Start sending command: %s", string(marshal))
 	response, err := mountClient.Post("http://unix", "application/json", bytes.NewReader(marshal))
 	if err != nil {
 		return status.Errorf(codes.Internal, "Failed to post command, err: %v", err)
 	}
 	defer response.Body.Close()
 	if response.StatusCode != http.StatusOK {
 		respBody, err := ioutil.ReadAll(response.Body)
+		log.Infof("start to mount bucket, cmd: %v", string(respBody))
```

Comment:
```suggestion
		log.Infof("start to mount bucket, cmd: %s", string(respBody))
```

Reply:
It can be changed, but it's not necessary